edb/ir/ast.py

```diff
@@ -721,10 +721,26 @@ class MutatingStmt(Stmt):
     subject: Set
 
 
+OnConflictElse = typing.NamedTuple(
```

Comment:
```suggestion
class OnConflictElse(typing.NamedTuple):
```

Reply:
Oh yeah! I'm still not used to getting to use python 3.6 features...

Reply:
;) 

Brace yourself as we'll start using 3.9 shortly after it's released.